primitives/chain-rococo/src/lib.rs

```diff
@@ -55,14 +55,14 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 /// See: https://github.com/paritytech/polkadot/blob/master/runtime/rococo/src/lib.rs
 #[derive(parity_scale_codec::Encode, parity_scale_codec::Decode, Debug, PartialEq, Eq, Clone)]
 pub enum Call {
-	/// Westend bridge pallet.
+	/// Generic bridge pallet.
```

Comment:
I left this enum and inner type in tact since I'm using it as part of the Wococo bridge